### PR TITLE
Iptables: update to 1.6.1

### PR DESCRIPTION
**Pull request by Ansuel**
Refreshed some patches and updated in support of newer kernel (4.9)

From the changelog page:
"This release includes accumulated fixes and enhancements for the
following matches:

* ah
* connlabel
* cgroup
* devgroup
* dst
* icmp6
* ipcomp
* ipv6header
* quota
* set
* socket
* string

and targets:

* CT
* REJECT
* SET
* SNAT
* SNPT,DNPT
* SYNPROXY
* TEE

We also got rid of the very very old MIRROR and SAME targets and the
unclean match, that were removed from the kernel tree long time ago.
We also got patches to update different aspects of our manpages."

With the help of Ralph Sennhauser <ralph.sennhauser@gmail.com> for some modification

Signed-off-by: Ansuel Smith <ansuelsmth@gmail.com>

### CODE REVIEW

**Comment by diizzyy** [comment]
Upstream is at 1.6.1, please update

**Comment by Ansuel** [comment]
@diizzyy done, now is compiling
(don't know the 2 times icon in the commit btw...)

**Comment by Ansuel** [comment]
I removed one mirror cause it doesn't had the new iptables source (it stops to 1.4.21)

**Comment by thagabe** [comment]
tmp/.config-package.in:51452: glob failed: No files found "package/network/utils/iptables/Config.in"
/home/thagabe/Git/WRT-Build/include/toplevel.mk:126: recipe for target 'menuconfig' failed
make: *** [menuconfig] Error 1

Created a new branch, pulled your PR, can't build

**Comment by Ansuel** [comment]
I have some problem with the updated version (1.6.1) i think today i will fix it

**Comment by Ansuel** [comment]
@diizzyy @thagabe i fixed it problem was with new xtables api, firewall3 need a patch to support it.

**Comment by thagabe** [comment]
Will build and test, thanks for your work!

**Comment by Ansuel** [comment]
@thagabe can you contact the mantainer of the firewall3 package so we don't need to use the extra patch?

**Comment by thagabe** [comment]
package 0001? I'll ask

**Comment by Ansuel** [comment]
Yep, up in the comments you have the reason of that patch

**Comment by Ansuel** [comment]
@diizzyy  any problem with this?

**Comment by diizzyy** [comment]
@Ansuel 
Wrong person to ask, I'll tag @jow- instead who's the maintainer.

**Comment by jow-** [comment]
@Ansuel - from a first look it seems fine to me but I am currently busy with other tasks. Will pick this once I did some before/after size comparisations.

**Comment by Ansuel** [comment]
Removed the extra patch for new xtables api.

**Comment by Ansuel** [comment]
@nbd168 can you check this too ? i currently use this and i don't have any problem

File: package/network/config/firewall/Makefile [hunk 1]
```diff
@@ -19,6 +19,8 @@ PKG_MIRROR_HASH:=7ee075f05977e5d9a78e661b537e6eb077c8f328ff2e71d1e2fbef44cca9735
 PKG_MAINTAINER:=Jo-Philipp Wich <jo@mein.io>
 PKG_LICENSE:=ISC
 
+PKG_FIXUP:=autoreconf
```
Comment (jow-) [inline package/network/config/firewall/Makefile]: firewall3 is `cmake` based so the fixup doesn ot apply to it

**Comment by Ansuel** [comment]
Should be good now 
@jow-

**Pull request closed without merging**